templates/appliance/community.html

```diff
@@ -1,216 +1,213 @@
 {% extends "appliance/_base-appliance.html" %}
 
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+{% from "_macros/vf_tiered-list.jinja" import vf_tiered_list %}
+
 {% block title %}Ubuntu Appliance Community - Governance and Policies{% endblock %}
 
-{% block meta_description %}Join the Ubuntu community by developing, discussing or even simply downloading one of the Ubuntu Appliances. Anyone can create an Ubuntu Appliance. Have your project benefit from all of the goodness that goes into Ubuntu.{% endblock %}
+{% block meta_description %}
+  Join the Ubuntu community by developing, discussing or even simply downloading one of the Ubuntu appliances. Anyone can create an Ubuntu appliance. Have your project benefit from all of the goodness that goes into Ubuntu.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit
+{% endblock meta_copydoc %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit{% endblock meta_copydoc %}
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
-<section class="p-strip--suru-topped">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h1>
-        Welcome to our Ubuntu Appliance community
-      </h1>
-      <p>
-        Whether you are a power user, a developer, a member of the Ubuntu project or a new contributor to Ubuntu Appliances, we invite you to join our Ubuntu Appliance Discourse and make your voice heard.
-      </p>
+
+  {% call(slot) vf_hero(
+    title_text='Welcome to our<br /> Ubuntu appliance community',
```

Comment:
The break makes this wrap a bit strangely on smaller screens
```suggestion
    title_text='Welcome to our<br class="u-hide--small"/> Ubuntu appliance community',
```